### ISSUE TITLE
Supported Multiplatform unit tests

### ISSUE THREAD

**Issue by qiaoyuang** [ctripcorp/SQLlin#8]
Now, SQLlin just supports macOS X64 unit tests and Android instrumentation tests. Next step, I will make the MinGW X64 and Linux X64 unit tests available. When I finished that, I will research if run other Apple(iOS, watchOS, tvOS) unit tests on simulator is available.

**Comment by qiaoyuang** [comment]
The [#11](https://github.com/ctripcorp/SQLlin/pull/11) was merged. Windows platform still has environment problem, It will be fixed, and Linux unit tests on CI/CD will be supported in the future.

**Comment by qiaoyuang** [comment]
The [#13](https://github.com/ctripcorp/SQLlin/pull/13) and [#14](https://github.com/ctripcorp/SQLlin/pull/14)  was merged. I fixed the link problem in Windows. Finally, I used the local sqlite.a file to be linked and give up use MSYS2. I also created a Linux environment job that use for build Linux artifact and run unit tests. And I added a lower version(Android 8) Android instrumented tests step in job of Linux environment.

I think I can close this issue now, but maybe I will do more optimization about CI/CD.
